### PR TITLE
docs: change the component test documentation to show how to change properties

### PR DESCRIPTION
**Pull request by doudou**

### CODE REVIEW

File: source/components/testing.html.md [hunk 1]
```diff
@@ -197,6 +209,10 @@ expect_execution { task.stop! }
     .to { EXPECTATIONS }
 ~~~
 
+If you want to test the start step, replace first `syskit_configure_and_start`
+by `syskit_and_configure`
```
Comment (wvmcastro) [inline source/components/testing.html.md:213]:
```suggestion
by `syskit_configure` followed by `syskit_start`
```
maybe?

Comment (doudou) [reply source/components/testing.html.md:213]: `syskit_configure_and_start` actually

Comment (doudou) [reply source/components/testing.html.md]: Corrected

File: source/components/testing.html.md [hunk 2]
```diff
@@ -197,6 +209,10 @@ expect_execution { task.stop! }
     .to { EXPECTATIONS }
 ~~~
 
+If you want to test the start step, replace first `syskit_configure_and_start`
+by `syskit_configure_and_start`
```
[the remark this answers is not in the document]
Comment (doudou) [reply source/components/testing.html.md:213]: Indeed not ... fixed

**Review by wvmcastro — APPROVED** [review]

**Pull request merged**